README.md

```diff
@@ -10,3 +10,27 @@ The included `Vagrantfile` simplifies VM creation, but doesn't make it so simple
 *Before running `vagrant up`*, be sure to patch `./SOLUTIONS/s_02/k8sMaster.sh` with the included patch file (this makes `kubeadm init` advertise on the subnet that the minion VM can see).
 
 This will take you through the otherwise painful parts of lab 2.2, and you'll pick up manually at the "Configure the Master Node" heading. Be sure to understand the steps described in those early parts of lab 2.2, though.
+
+*CEP Notes:*
+  * The following will add the vagrant user to the docker group and allow accessing the commands from the vagrant user:
+```
+sudo usermod -aG docker $USER
+source ~/.bashrc
```

Comment:
what's this line for?

Reply:
Forces the user session to reload without having to log out and back in.